SuperBuild/External_OpenCV.cmake

```diff
@@ -26,13 +26,13 @@ if(NOT DEFINED OpenCV_DIR AND NOT ${CMAKE_PROJECT_NAME}_USE_SYSTEM_${proj})
 
   ExternalProject_SetIfNotDefined(
     ${CMAKE_PROJECT_NAME}_${proj}_GIT_REPOSITORY
-    "https://github.com/opencv/opencv.git"
+    https://github.com/Slicer/opencv.git
     QUIET
     )
 
   ExternalProject_SetIfNotDefined(
     ${CMAKE_PROJECT_NAME}_${proj}_GIT_TAG
-    "3.4.6"
+    3.4.6_user_release_python
```

Comment:
Please, rename branch as `slicer-3.4.7-2019.07.25-4a7ca5a29` and specify the commit hash instead of the name of the branch.

It should be:

```
c8098f6c4001fd1a3a8f599f959beb3833f737b6  # slicer-3.4.7-2019.07.25-4a7ca5a29
```

See https://www.slicer.org/wiki/Documentation/Nightly/Developers/ProjectForks#Naming_convention_for_Slicer_fork_branches